R/expand.R

```diff
@@ -1,7 +1,7 @@
 #' Expand messy dates to lists of dates
 #'
 #' These functions expand on date ranges, sets of dates, and unspecified or
-#' approximate dates (annotated with '..', '{ , }', 'XX' or '~').
+#' approximate dates (annotated with '..', '{}', 'XX' or '~').
```

Comment:
Why not clarify that a set will be separated by commas?

Reply:
This causes issues with documentation in RHUB and other tests and if I escape the coma, the documentation appears separated for each bracket...

Reply:
Bon